app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -212,21 +159,28 @@ private fun ConversationScreenContent(
     onSendAttachment: (AttachmentBundle?) -> Unit,
     onDownloadAsset: (String) -> Unit,
     onImageFullScreenMode: (String) -> Unit,
-    onMessageComposerError: (ConversationErrors) -> Unit,
+    onMessageComposerError: (ConversationSnackbarMessages) -> Unit,
     conversationState: ConversationViewState,
     conversationScreenState: ConversationScreenState
 ) {
     val lazyListState = rememberLazyListState()
     val coroutineScope = rememberCoroutineScope()
+    val context = LocalContext.current
 
     LaunchedEffect(conversationState.messages) {
         lazyListState.animateScrollToItem(0)
     }
 
-    conversationState.onError?.let { errorCode ->
-        val errorMessage = getErrorMessage(errorCode)
-        LaunchedEffect(conversationState.onError) {
-            conversationScreenState.snackBarHostState.showSnackbar(errorMessage)
+    conversationState.onSnackbarMessage?.let { msgCode ->
```

Comment:
minor thing ;p but since we are using message word here and there, could we rename it to messageCode ?